app/pages/home.cjsx

```diff
@@ -41,46 +64,64 @@ module.exports = React.createClass
   componentWillUnmount: ->
     document.documentElement.classList.remove 'on-home-page'
 
+  lastFourProjects: ->
+    @props.user.get("project_preferences", page_size: 4, sort: '+updated_at')
+
   render: ->
     aboutItems = ['contribute', 'explore', 'collaborate', 'discover']
 
     <div className="home-page">
       <section className="hero on-dark">
         <ZooniverseLogoType />
-        <h3 className="hero-title"><Translate content="home.hero.title" /></h3>
-        <p className="hero-tagline"><Translate content="home.hero.tagline" /></p>
-        <Link to="projects" className="call-to-action standard-button hero-button x-large"><Translate content="home.hero.button" /></Link>
-      </section>
-      <section className="about-zooniverse">
-        <div className="about-items-list">
-          {for item in aboutItems
-            <div key={item} className="about-item">
-              <div className="about-item-wrapper">
-                <img className="about-image" src="./assets/home-#{item}.gif" alt="" />
-                <div className="about-item-content">
-                  <Translate component="h5" content="home.about.#{item}.title" />
-                  <Markdown>{counterpart "home.about.#{item}.content"}</Markdown>
-                </div>
+        {if @props.user
```

Comment:
Could this be dependent upon if both a user exists and they've actually classified on something?